Ziptastic/src/main/java/Ziptastic_Object/Ziptastic.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package Ziptastic_Object;
+
+import java.io.IOException;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+
+
+public class Ziptastic {
+    
+    private static String url = "https://zip.getziptastic.com/v3/US/";
+    private static String apiKey;
+    
+    public Ziptastic(String key) {
+       this.apiKey = key; 
+    }
+    
+    
+    public static String ForwardGeocoding(int zipcode) throws IOException{

```

Comment:
1. ForwardGeocode() would probably be a better name. I feel as if GetFromPostalCode() is a little lengthy, but that may just be a matter of personal preference. But at the same time, it would be easier for someone that doesn't fully understand the geocoding terminology to pickup on the object use.

2. Yes, that wouldn't be an issue, its simply reconstructing a similar method to accept a string input.